frontend/lib/resolution/disambiguation.cpp

```diff
@@ -1096,6 +1370,255 @@ moreVisible(const DisambiguationContext& dctx,
                           fn1Id, fn2Id);
 }
 
+/**
+  Compare two argument mappings, given a set of actual arguments, and set the
+  disambiguation state appropriately.
+
+  This function implements the argument mapping comparison component of the
+  disambiguation procedure as detailed in section 13.14.3 of the Chapel
+  language specification (page 107).
```

Comment:
Ehh.. these section numbers and pages are out of date. We don't even have section numbers anymore. Let's just put "as detailed in the language specification".